coderefs/coderefs.go

```diff
@@ -108,8 +113,13 @@ func calculateStaleBranches(branches []ld.BranchRep, remoteBranches map[string]b
 
 func handleOutput(opts options.Options, matcher search.Matcher, branch ld.BranchRep, repoParams ld.RepoParams, ldApi ld.ApiClient) {
 	outDir := opts.OutDir
+	projectKeys := make([]string, 1)
+	for _, project := range opts.Projects {
+		projectKeys = append(projectKeys, project.ProjectKey)
+	}
+
 	if outDir != "" {
-		outPath, err := branch.WriteToCSV(outDir, opts.ProjKey, repoParams.Name, opts.Revision)
+		outPath, err := branch.WriteToCSV(outDir, repoParams.Name, opts.Revision)
```

Comment:
As noted in PR description. We no longer include the `projKey` in the output file name so this is a breaking change.